ta/pkcs11/src/pkcs11_token.h

```diff
@@ -147,7 +147,7 @@ struct active_processing {
 	uint32_t mecha_type;
 	bool always_authen;
 	bool relogged;
-	bool updated;
+	enum processing_step op_step;
```

Comment:
this field should be explicitly initialized to `PKCS11_FUNC_STEP_INIT` from `set_processing_state()` where `struct active_processing` is allocated. It happens that the struct is zero-filled and that `PKCS11_FUNC_STEP_INIT` equals 0, but I would prefer an explicit init:
```patch
 	proc = TEE_Malloc(sizeof(*proc), TEE_MALLOC_FILL_ZERO);
 	if (!proc)
 		return PKCS11_CKR_DEVICE_MEMORY;
 
 	/* Boolean are default to false and pointers to NULL */
 	proc->state = state;
+	proc->step = PKCS11_FUNC_STEP_INIT;
 	proc->tee_op_handle = TEE_HANDLE_NULL;
```

Also, I would maybe rename the field to `proc_step` or simply `step` and move it right below field `state` at line 146.

Reply:
Done, changed to step and initialised the variable